### DISCUSSION TITLE
Make menu entry swapper have different left click options for PoH fairy ring/spirit tree and mainland fairy rings

### DISCUSSION THREAD

**Discussion by akrueger47** [runelite/runelite#14966] [Ideas]
As is, if you have a PoH Fairy ring/Spirit tree combo, you cant have left click for tree in your house while also toggling the option for fairy rings to be left click configure. The fairy ring setting overrides left click tree in your PoH, and if you turn off the fairy ring setting, fairy rings out in the world default to left click zanaris. Changing the left click to configure also changes the PoH back to configure instead of tree.

Would be nice to be able to set left click on poh tree/ring to something different than mainland fairy rings

**Comment by geheur** [comment]
People have asked for this before: https://github.com/runelite/runelite/issues/14864
